api/v1alpha1/lvmcluster_types.go

```diff
@@ -79,9 +79,27 @@ type LVMClusterStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// ready describes if the LvmCluster is ready.
+	// Ready describes if the LVMCluster is ready.
 	// +optional
 	Ready bool `json:"ready,omitempty"`
+	// DeviceClassStatus describes the status of the VG on each node
+	DeviceClassStatuses []DeviceClassStatus `json:"deviceClassStatuses,omitempty"`
+}
+
+// VGNodeStatus defines the observed status of the VG across all nodes
+type DeviceClassStatus struct {
+	// Name is the name of the VG
```

Comment:
This might cause confusion between `VG` and `DeviceClass`.   I think we should use either `VG` or `DeviceClass`